controllers/users.controller.js

```diff
@@ -52,9 +52,10 @@ exports.getPortfolioByUsername = (req, res, next) => {
 		});
 };
 
-exports.emptyPortfolio = (req, res, next) => {
+exports.updatePortfolio = (req, res, next) => {
 	const { username, portfolio } = req.params;
-	removePortfolioData(username, portfolio)
+	const tickers = req.body;
```

Comment:
I believe tickers would need to be deconstructed to { tickers } = req.body on line 57 as the body that is being sent over is an object with the key of "tickers" and a variable of "tickerArr" which is what you actually want